mule-module-apikit/src/test/resources/org/mule/module/apikit/console/console.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mule xmlns="http://www.mulesoft.org/schema/mule/core"
+      xmlns:apikit="http://www.mulesoft.org/schema/mule/apikit"
+      xmlns:http="http://www.mulesoft.org/schema/mule/http"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xsi:schemaLocation="http://www.mulesoft.org/schema/mule/http http://www.mulesoft.org/schema/mule/http/current/mule-http.xsd
+      http://www.mulesoft.org/schema/mule/apikit http://www.mulesoft.org/schema/mule/apikit/current/mule-apikit.xsd
+      http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd">
+
+    <configuration defaultErrorHandler-ref="defaultErrorHandler"/>
+
+    <http:listener-config name="httpListenerConfigDefault">
+        <http:listener-connection host="localhost" port="${serverPort}"/>
+    </http:listener-config>
+
+    <http:listener-config name="httpListenerConfigForInvalidConsoleFlow">
+        <http:listener-connection host="localhost" port="${serverPort2}"/>
+    </http:listener-config>
+
+
+    <apikit:config name="apikitConfig" raml="org/mule/module/apikit/simple-routing/simple.raml"
+                   outboundHeadersMapName="outboundHeaders" httpStatusVarName="httpStatus">
+    </apikit:config>
+
+    <flow name="api-console">
+        <http:listener config-ref="httpListenerConfigDefault" path="/console/*">
+            <http:response statusCode="#[variables.httpStatus default 200]"><http:headers>#[variables.outboundHeaders]</http:headers></http:response>
+            <http:error-response statusCode="#[variables.httpStatus default 500]"><http:headers>#[variables.outboundHeaders]</http:headers></http:error-response>
+        </http:listener>
+
+        <apikit:console config-ref="apikitConfig" />
+    </flow>
+
+    <flow name="invalid-api-console">
+        <http:listener config-ref="httpListenerConfigForInvalidConsoleFlow" path="/konsole/"/>
+        <apikit:console config-ref="apikitConfig" />
+    </flow>
+
+
+    <error-handler name="defaultErrorHandler">
```

Comment:
Delete this error handler, and this line: 
configuration defaultErrorHandler-ref="defaultErrorHandler"